src/components/views/PCI.js

```diff
@@ -107,6 +108,7 @@ export default class PCI extends React.Component {
               <PCICoverage data={data} />
               <Agreements
                 data={data}
+                handlers={handlers}
```

Comment:
Do we need to pass all the handlers here? Instead you could just have
`onNeedMoreEntitlements={handlers. onNeedMoreEntitlements}`

Reply:
Agreements is expecting handlers, it's used elsewhere